firebase-firestore/src/jsMain/kotlin/dev/gitlive/firebase/firestore/firestore.kt

```diff
@@ -380,6 +392,11 @@ actual enum class FirestoreExceptionCode {
     UNAUTHENTICATED
 }
 
+actual enum class Direction {
+    ASCENDING,
+    DESCENDING
+}
+
 inline fun <T, R> T.rethrow(function: T.() -> R): R = dev.gitlive.firebase.firestore.rethrow { function() }
```

Comment:
could add internal val name: String to Direction instead of having if (direction == Direction.ASCENDING) "asc" else "desc") above